config.py

```diff
@@ -137,8 +137,7 @@ class TestingConfig(BaseConfig):
     MOCK_EMAIL = True
 
     # Use in-memory SQLite database for testing
-    SQLALCHEMY_DATABASE_URI = "postgresql:///bit_schema_test"
-    # SQLALCHEMY_DATABASE_URI = BaseConfig.build_db_test_uri()
+    SQLALCHEMY_DATABASE_URI = "postgresql://postgres:postgres@localhost:5432/bit_schema_test"
```

Comment:
can't we use/uncomment line 141 below for this instead of changing the code? since the original 140 is the syntax for MacOS. I'll check it out once I merge this to the PR branch.